skimage/util/dtype.py

```diff
@@ -191,19 +207,19 @@ def _scale(a, n, m, copy=True):
         imax_in = np.iinfo(dtype_in).max
 
     if kind_in == 'f':
-        if np.min(image) < -1.0 or np.max(image) > 1.0:
+        if dc.eager_min(image) < -1.0 or dc.eager_min(image) > 1.0:
```

Comment:
You don't need a special eager version of min here -- dask arrays are eagerly evaluated when you call their `__bool__` method, so it happens automatically in `if` statements.